### DISCUSSION TITLE
How to fetch audio segments in real time from diarization pipeline

### DISCUSSION THREAD

**Discussion by Yagna24** [pyannote/pyannote-audio#874] [Q&A]
Is there a way around to get the discrete audio segments from the diarization. I need to store the segments.

**Comment by Anticope12** [comment]
Hi @Yagna24 , Were you able to solve this ?

**Reply by Yagna24** [reply]
Yes @Anticope12 , what you can do is, 

### Load and apply diarization
```
pipeline = torch.hub.load('pyannote/pyannote-audio', 'dia_ami')
diarized = pipeline(audio_input)
```
### Get contents out of the diarized audio that contains the timestamps and labels
`segments = diarized.for_json()["content"]`
